facts/4.3/windows-10-x86_64.facts

```diff
@@ -1,120 +1,88 @@
 {
   "aio_agent_version": "7.24.0",
-  "architecture": "x64",
-  "dhcp_servers": {
-    "Ethernet": "168.63.129.16",
-    "system": "168.63.129.16"
-  },
   "dmi": {
-    "manufacturer": "American Megatrends Inc.",
+    "manufacturer": "innotek GmbH",
     "product": {
-      "name": "Virtual Machine",
-      "serial_number": "0000-0009-6402-6941-5339-5438-01",
-      "uuid": "85C91DF4-F860-4CF9-BEEB-2371A5318681"
+      "name": "VirtualBox",
+      "serial_number": "0",
+      "uuid": "82E587EA-9C5A-41D2-B96A-D58BAF6CC890"
     }
   },
-  "domain": "example.com",
   "env_windows_installdir": "C:\\Program Files\\Puppet Labs\\Puppet",
   "facterversion": "4.3.1",
   "fips_enabled": false,
   "fqdn": "foo.example.com",
-  "hardwareisa": "x64",
-  "hardwaremodel": "x86_64",
-  "hostname": "foo",
   "hypervisors": {
-    "hyperv": {
+    "virtualbox": {
+      "revision": "161095",
+      "version": "7.0.14"
     }
   },
-  "id": "foo\\windows",
   "identity": {
     "privileged": true,
-    "user": "foo\\windows"
+    "user": "foo\\vagrant"
   },
-  "interfaces": "Ethernet",
-  "ipaddress": "10.138.1.6",
-  "ipaddress6": "fe80::5111:281c:6d79:d8c5",
-  "ipaddress6_Ethernet": "fe80::5111:281c:6d79:d8c5",
-  "ipaddress_Ethernet": "10.138.1.6",
   "is_virtual": true,
   "kernel": "windows",
   "kernelmajversion": "10.0",
   "kernelrelease": "10.0.19045",
   "kernelversion": "10.0.19045",
-  "macaddress": "60:45:BD:D0:87:D9",
-  "macaddress_Ethernet": "60:45:BD:D0:87:D9",
-  "manufacturer": "American Megatrends Inc.",
   "memory": {
     "system": {
-      "available": "12.52 GiB",
-      "available_bytes": 13447675904,
-      "capacity": "21.72%",
-      "total": "16.00 GiB",
-      "total_bytes": 17179398144,
-      "used": "3.48 GiB",
-      "used_bytes": 3731722240
+      "available": "2.09 GiB",
+      "available_bytes": 2244268032,
+      "capacity": "47.54%",
+      "total": "3.98 GiB",
+      "total_bytes": 4277866496,
+      "used": "1.89 GiB",
+      "used_bytes": 2033598464
     }
   },
-  "memoryfree": "12.52 GiB",
-  "memoryfree_mb": 12824.703125,
-  "memorysize": "16.00 GiB",
-  "memorysize_mb": 16383.55078125,
-  "mtu_Ethernet": 1500,
-  "netmask": "255.255.255.0",
-  "netmask6": "ffff:ffff:ffff:ffff::",
-  "netmask6_Ethernet": "ffff:ffff:ffff:ffff::",
-  "netmask_Ethernet": "255.255.255.0",
-  "network": "10.138.1.0",
-  "network6": "fe80::",
-  "network6_Ethernet": "fe80::",
-  "network_Ethernet": "10.138.1.0",
   "networking": {
-    "dhcp": "168.63.129.16",
-    "domain": "example.com",
-    "fqdn": "foo.example.com",
+    "dhcp": "10.0.2.2",
+    "domain": "exmaple.com",
+    "fqdn": "foo.exmaple.com",
```

Comment:
This breaks at least one test and seems like a typo, which is wierd since it is fully autogenerated

Reply:
Woops, that was my typo. Regenerating...